README.md

```diff
@@ -54,42 +54,30 @@ Currently the easiest way to install Home Manager is as follows:
     NixOS you can control this option using the
     [`nix.allowedUsers`][nixosAllowedUsers] system option.
 
-2.  Assign a temporary variable holding the URL to the appropriate
-    archive. Typically this is
+2.  Add the appropriate Home Manager channel. Typically this is
 
     ```console
-    $ HM_PATH=https://github.com/rycee/home-manager/archive/master.tar.gz
+    $ nix-channel --add https://github.com/rycee/home-manager/archive/master.tar.gz home-manager
     ```
 
     if you are following Nixpkgs master or an unstable channel and
 
     ```console
-    $ HM_PATH=https://github.com/rycee/home-manager/archive/release-18.03.tar.gz
+    $ nix-channel --add https://github.com/rycee/home-manager/archive/release-18.09.tar.gz home-manager
     ```
 
-    if you follow a Nixpkgs version 18.03 channel.
+    if you follow a Nixpkgs version 18.0339 channel.
```

Comment:
This should be 18.09 right?

Reply:
Yeah, that's right. We haven't yet gotten to month 339 😀 Thanks for noticing!